lib/config.js

```diff
@@ -1,26 +1,27 @@
-const { version } = require('../package.json');
```

Comment:
Half the colons are single half double. Dont we use single for everything except sting interpolation?

Reply:
Yeah, sorry — think my linter is misconfigured. Will fix